test/integration/commands/validate_data/test_reformat_validaton_result.py

```diff
@@ -3,9 +3,11 @@
 import pytest
 from rdflib import Graph
 
+from dsp_tools.commands.validate_data.models.input_problems import ContentRegexViolation
```

Comment:
I have not previously tested the value type and content, because I need the data as ttl too. Otherwise I get a problem with the bn IDs that are different. I did not want to maintain two test data files. But now I think we can easily move the assertions to e2e tests so we can in the long term get rid of the files. For the meantime it gives us some security.

Reply:
I understand. The good thing is that you have already "modularized" the test data into separate files.